words/words.md

```diff
@@ -947,57 +951,57 @@ Fortunatamente, dato che l'estate riduce R, la situazione migliorerà:
 
 Oh.
 
-Al contrario di quanto si pensi, l'estate rende i picchi peggiori *e* regolari! Questo perché l'estate riducei i nuovi <span class="nowrap"><icon i></icon>s,</span> che di conseguenza riduce i nuovi <span class="nowrap"><icon r></icon></span> immuni. Questo significa che l'immunità precipita in estate, *creando* larghi picchi regolari in inverno.
+In maniera abbastanza controintuitiva, l'estate rende i picchi peggiori *e* regolari! Questo perché l'estate riduce il numero dei nuovi <span class="nowrap"><icon i></icon>s,</span> che di conseguenza riduce quello dei nuovi <span class="nowrap"><icon r></icon></span> immuni. Questo significa che l'immunità precipita in estate, *creando* notevoli picchi regolari in inverno.
```

Comment:
Ok, I think I already commented on this part and it changed again.  Probably this alredy fix that...